--- truecrypt-7.0-source/Makefile.orig	2010-09-05 16:32:10.000000000 +0200
+++ truecrypt-7.0-source/Makefile	2010-12-09 15:49:23.517359765 +0100
@@ -87,7 +87,7 @@ endif
 
 ifeq "$(TC_BUILD_CONFIG)" "Release"
 
-	C_CXX_FLAGS += -O2 -fno-strict-aliasing  # Do not enable strict aliasing
+	C_CXX_FLAGS += $(RPM_OPT_FLAGS) -fno-strict-aliasing  # Do not enable strict aliasing
 	export WX_BUILD_DIR ?= $(BASE_DIR)/wxrelease
 	WX_CONFIGURE_FLAGS += --disable-debug_flag --disable-debug_gdb --disable-debug_info
 
